.travis.yml

```diff
@@ -0,0 +1,25 @@
+language: cpp
```

Comment:
Sorry if I was unclear.  I meant that a comment should be added inline in the file itself.  The goal is for any future developer to be able to look at the source file and know its purpose.